source/assets/styles/recipe-add.css

```diff
@@ -303,7 +303,7 @@ input {
 }
 
 .input-list {
-	margin-left: 25px;
+	margin-left: 35px;
```

Comment:
fIX the issue if the instruction or ingredients number go 2 digits then it will go out of the box and got cutoff in phone model.